src/originalBudgetTrackingApp/export/exportTransactions.ts

```diff
@@ -31,7 +30,9 @@ export async function createTransactionsInExternalVendors(
         throw e;
       }
     }
-  }
+  });
+
+  await Promise.all(exportPromises);
```

Comment:
I wanted to ask you to return the `executionResult` from the promise, instead of updating the `executionResults` from the closure.

But in that case, if one of the promises will fail, you will not get any result, even from the successes promises.

Anyway, have you tried [`Promise.allSettled()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled)? I heard about but never tried...

(This is also just comment on my opinion about nodeJS code)

Reply:
Thats a good point. Our current behaviour is that if one of them fails they all fail and it would be better to be able to allow all to run and fail only what failed. This will also require the ui to show this correctly.

This is a small and focused pr so I don't want to mix this in